### PR TITLE
Improve contrast on Docsearch dark mode for selected item

### LINKED ISSUE #1324: Algolia dark mode highlighted result has low contrast
Hello, while browsing the site in dark mode I observed highlighted results in the Algolia search modal have white text on a bright orange background, which has very low contrast as you'll see below:

image

To meet WCAG AAA standards, an easy win would be choosing a near-black text colour for highlighted results.
I suggest:

- `#ff995f` text on a `#141415` background (note this orange is different from the `#ff7e33` orange used throughout the site)
- `#141415` text against the orange background

### PR DESCRIPTION
**Pull request by charlesfrisbee**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)

#### Description

- Closes #1324
Algolia Docsearch highlighted item now has dark text for better contrast against orange compared to previous white color.

Screen Shot 2022-08-22 at 23 42 38

### CODE REVIEW

**Comment by sarah11918** [comment]
@charlesfrisbee This looks good! I did just compare it to the docs site live, and yes, I see an improvement! I'm going to ask @Yan-Thomas to review this one and take it from here. Thank you so much for this contribution!

File: public/theme.css [hunk 1]
```diff
@@ -195,6 +195,7 @@
 	--docsearch-modal-shadow: 0 0 var(--theme-glow-blur) var(--theme-glow-diffuse), inset 0 0 0 1px var(--theme-glow-highlight),
 		inset 0 0 var(--theme-glow-blur) var(--theme-glow-diffuse);
 	--docsearch-container-background: hsla(var(--color-gray-10), 0.8);
+	--docsearch-hit-active-color: var(--theme-bg);
```
Comment (Yan-Thomas) [inline public/theme.css:198]: I'd suggest an even darker color so it has better contrast in greyscale and in smaller font sizes. What do you think? (Same color we use for the main text in the light theme)
```suggestion
	--docsearch-hit-active-color: hsla(var(--color-gray-10), 1);
```

**Review by Yan-Thomas — COMMENTED** [review]
Thanks for this @charlesfrisbee, really is an improvement! I added a suggestion to make it even better 🙌

File: public/theme.css (hunk 1 above)
Comment (charlesfrisbee) [reply public/theme.css]: Yep I agree @Yan-Thomas, looks better thanks!

**Comment by Yan-Thomas** [comment]
LGTM, thanks @charlesfrisbee!

**Pull request merged**